### DISCUSSION TITLE
GPU and parallel computaion

### DISCUSSION THREAD

**Discussion by skababji** [vanderschaarlab/synthcity#254] [Q&A] [answered]
Does the library support parallel GPU computation when training the neural networks. Is there a way to enforce that?

**Comment by stefanosh** [comment]
Would be interested on this, any updates?

**Reply by skababji** [reply, marked as answer]
In your code, define multiple processes and direct each process to a specific GPU using torch.cuda.set_device(<your_gpu_index>)
